### ISSUE TITLE
Файл G2-105.sty

### ISSUE THREAD

**Issue by einhander** [rominf/latex-g7-32#87]
У нас сейчас два основных стилевых файла G2-105.sty и G7-32.sty. Если мы сейчас занимаемся рефакторингом кода имеет смысл перенести весь функционал из G2-105.sty в G7-32.sty.

**Comment by dvarubla** [comment]
Конечно, так будет лучше. У нас сейчас ещё GostBase.clo есть, может и его туда же? Кстати, я похоже забыл отредактировать опции в cls файле.

**Comment by einhander** [comment]
>  GostBase.clo есть, может и его туда же?

Ага
